Java/src/main/java/com/gildedrose/Item.java

```diff
@@ -18,4 +18,32 @@ public Item(String name, int sellIn, int quality) {
    public String toString() {
         return this.name + ", " + this.sellIn + ", " + this.quality;
     }
+
+    public boolean hasQuality() {
```

Comment:
you could change the visibility to private if you only changing the values with these setters